.github/workflows/pull-request.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
             capabilities.json
 
   go-build:
-    name: Go Build (${{ matrix.os }}${{ matrix.arch && format(' {0}', matrix.arch) || '' }})
+    name: Go Build (${{ matrix.os }}${{ matrix.arch && format(' {0}', matrix.arch) || '' }}${{ matrix.go_tags }})
```

Comment:
What happens in the case `go_tags` is not set?

Reply:
It'll end up as nothing, I think. The CI run of this PR shows it. The only problem is that we seem to be unsetting some args what we need, too: https://github.com/open-policy-agent/opa/actions/runs/7802363034/job/21279888203

Reply:
Yes, it will just be empty. Adjusted the call so that it won't interfere with other tags.

Reply:
@srenatus does this address your concern?

Reply:
Tests pass, so I'm happy 😃 Thanks!